vexim/config/functions.php

```diff
@@ -28,7 +28,7 @@ function validate_password($clear,$vclear)
      */
     function alias_validate_password($clear,$vclear) 
     {
-        return ($clear == $vclear);
+        return ($clear === $vclear);
```

Comment:
I understand I didn't mention this before, but could we perhaps merge these functions into one, with a third param indicating whether or not empty password is acceptable (`false` by default)?